getting_started/dedicated_administrators.adoc

```diff
@@ -140,6 +140,31 @@ If this group is currently empty or if you need assistance editing group
 membership, open a support case on the
 link:https://access.redhat.com/support/[Red Hat Customer Portal].
 
+[[managing-dedicated-readers-group]]
+== Managing the Dedicated Readers Group
+
+Users in the *dedicated-readers* group are granted view-only access to
+*dedicated-readers* projects.
+
+To view a list of current dedicated readers by user name, you can use the
+following command:
```

Comment:
The below two `oc adm` commands only can be run successfully for the user who has `dedicated-admin` or higher authority. I am not sure whether needs add some words at here, it is up to your decision.